### PR TITLE
[o2femtodream] First prototype analysis task for track-track femto

### PR DESCRIPTION
**Pull request by a-mathis**
- Introducing
  o PairCleaner - clean up clones
  o CutCulator - resolves the bit-wise encoding of syst. vars.
  o further improvements and bug fixes
  o plenty of documentation

### CODE REVIEW

File: Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDerived.h [hunk 1]
```diff
@@ -24,62 +24,72 @@
 
 namespace o2::aod
 {
+
+/// FemtoDreamCollision
 namespace femtodreamcollision
 {
-DECLARE_SOA_COLUMN(MultV0M, multV0M, float);
-DECLARE_SOA_COLUMN(Sphericity, sphericity, float);
+DECLARE_SOA_COLUMN(MultV0M, multV0M, float);       ///< V0M multiplicity
```
Comment (jgrosseo) [inline Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDerived.h]: The syntax to be put automatically in our documentation for the comments is : //! COMMENT

Comment (jgrosseo) [reply Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDerived.h]: Also below...

**Comment by a-mathis** [comment]
@jgrosseo @iarsene merge please

**Review by jgrosseo — APPROVED** [review]

**Comment by jgrosseo** [comment]
Thanks @a-mathis and all the best! :)

**Pull request merged**